packages/react-scripts/utils/createJestConfig.js

```diff
@@ -20,8 +20,8 @@ module.exports = (resolve, rootDir, isEjecting) => {
   const config = {
     collectCoverageFrom: ['src/**/*.{js,jsx}'],
     moduleNameMapper: {
-      '^.+\\.(?!(js|jsx|css|json)$)[^\\.]+$': resolve('config/jest/FileStub.js'),
```

Comment:
Hi @nwolverson. The first line said, match all file extensions except these few (`.js`, `.jsx`, `.css` & `.json`), and the second line said match these specific extensions (`.css`, `.less`, `.scss`, `.sass` & `styl`), yet the first line precluded the second line from running for all but one of the file types we were interested in (`.css`).

I could have added `.less`, `.scss`, `.sass` & `styl` to that line, but that just felt like repeating myself unnecessarily when we could instead make a catch-all rule like this the last one in the list.